public/js/always.js

```diff
@@ -117,11 +123,7 @@ function Dialog(id) {
       return console.warn('Trying to trigger not rendered dialog!');
     }
 
-    if (this.checkSession()) {
-      return this.element().classList.add(dialogClasses.active);
-    }
-   
-    return false;
+    return this.element().classList.add(dialogClasses.active);
```

Comment:
I don't think adding `active` class in the Check function is the best way to handle this stuff. And i think this function is also called in `open` function.